include/cudaq/Optimizer/Transforms/Passes.h

```diff
@@ -36,10 +36,12 @@ createApplyOpSpecializationPass(bool computeActionOpt);
 std::unique_ptr<mlir::Pass> createDelayMeasurementsPass();
 std::unique_ptr<mlir::Pass> createExpandMeasurementsPass();
 std::unique_ptr<mlir::Pass> createLambdaLiftingPass();
+std::unique_ptr<mlir::Pass> createLiftArrayAllocPass();
 std::unique_ptr<mlir::Pass> createLowerToCFGPass();
 std::unique_ptr<mlir::Pass> createObserveAnsatzPass(std::vector<bool> &);
 std::unique_ptr<mlir::Pass> createQuakeAddMetadata();
 std::unique_ptr<mlir::Pass> createQuakeAddDeallocs();
+std::unique_ptr<mlir::Pass> createStatePreparation();
```

Comment:
```suggestion
```
Use `createPrepareState()` which is already declared and defined. (If you prefer `createStatePreparation()` (I sort of do), change the name of the pass in Passes.td accordingly.

Reply:
Renamed, thanks!